invoicing/tests/test_utils.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+
+from invoicing.utils import generate_reference_number
+
+
+@pytest.mark.parametrize(
+    "identifier, expected",
+    [
+        (118, "28251187"),
+        ("123456", "28251234568"),
+        (999, "28259998"),
```

Comment:
Wasn't the problem case such that the check digit is 0? None of these has check digit of zero.

Reply:
118 would give that issue without the fix, you mean that I should add a case where it ends in 0?